test/util/all_close_f.cpp

```diff
@@ -238,6 +238,11 @@ ::testing::AssertionResult
     {
         return ::testing::AssertionFailure() << "a.size() != b.size() for all_close_f comparison.";
     }
+    if (a.size() == 0)
+    {
+        msg << "No elements to compare";
```

Comment:
Nothing being done w/ msg in this path.
Either use it like this:.
`return ::testing::AssertionSuccess() << msg.str();`
or skip it like this:
`return ::testing::AssertionSuccess() << "No elements to compare";`